app/views/index.html

```diff
@@ -24,147 +24,19 @@ <h1>NHS.UK redesign project</h1>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__development">Development</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td><a href="/styles/video">Video</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__development">Development</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td><a href="/styles/review-date">Page review date</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__development">Development</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td><a href="/styles/multi-page-nav">Multi-page nav</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__design">Design</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td>Homepage</td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__design">Design</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td>Health A-Z</td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__design">Design</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td><a href="/styles/phase-banner">Phase banner (This service is new and feedback)</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__development">Development</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td><a href="/styles/survey-banner">Survey banner</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__development">Development</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td><a href="/styles/emergency-banner">Emergency banner</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__development">Development</span></td>
-                        <td></td>
-                      </tr>
-                      <tr>
-                        <td><a href="/styles/cookie-banner">Cookie banner</a></td>
-                        <td class="td__status"><span class="nhsuk-tag nhsuk-tag__design">Design</span></td>
-                        <td></td>
-                      </tr>
-                      </tbody>
-                    </table>
+                  <h2>Prototypes</h2>
+                  <ul>
+                    <li><a href="#"> Prototype 1</a></li>
```

Comment:
Would it be better to use the names we gave the prototypes? Or add a description? I've no idea which prototype is 1, 2, or 3. But, if we as a team, have already defined and I've overlooked it somehow ignore me.

Reply:
It's a placeholder and we can pick that up after.